src/repository/storage/postgres/orm/sequelize/note.ts

```diff
@@ -231,6 +231,7 @@ export default class NoteSequelizeStorage {
         model: this.settingsModel,
         as: 'noteSettings',
         required: true,
+        attributes: [],
```

Comment:
what does this line do?

Reply:
This line removes noteSettings from returned Note object